content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,471 @@
+Many of the e-Invoicing features are implemented using Odoo's tax system. As such, it is very
+important that taxes are properly configured in order to correctly generate invoices and handle
+other billing use cases. For example, specific configurations are required for the **reverse
+charge** type of taxes. In case of a **reverse charge** tax, the seller does *not* charge the
+customer for the VAT but instead the customer pays the VAT *themselves* to their government. There
+are **two** main types:
 
-.. image:: italy/sending.png
+- :ref:`external reverse charge <italy/external-reverse>`;
+- :ref:`internal reverse charge  <italy/internal-reverse>`.
+
+.. _italy/external-reverse:
+
+External reverse charge
+-----------------------
+
+Invoices
+~~~~~~~~
+
+To make an export invoice, make sure that the invoice lines all use a tax configured for **reverse
+charge**. The **Italian** localization contains an **example** of a reverse charge tax for export in
+the EU to be used as reference (`0% EU`, invoice label `00eu`), which can be found under
+:menuselection:`Accounting --> Configuration --> Taxes`. Exports are exempt from VAT, and therefore
+**reverse charge** taxes require the :guilabel:`Has exoneration of tax (Italy)` option ticked, with
+both the :guilabel:`Exoneration` kind and :guilabel:`Law Reference` filled in.
+
+.. image:: italy/italy-tax.png
    :align: center
-   :alt: Asynchronous sending on a customer invoice
+   :alt: External reverse charge settings
+
+.. note::
+   If you need to use a different kind of :guilabel:`Exoneration`, click :menuselection:`Action -->
+   Duplicate` within the tax menu to create a copy of an existing similar tax. Then, select another
+   :guilabel:`Exoneration`, and :guilabel:`Save`. Repeat this process as many times as you need
+   different kind of :guilabel:`Exoneration` taxes.
+
+.. tip::
+   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
+   differentiate them easily.
+
+On your invoice, select the corresponding tax you need in the :guilabel:`Taxes` field. You can find
+the following **additional info** by opening the **XML** file of the issued invoice:
+
+- :guilabel:`SdI address (Codice Destinatario)`: must be filled for both **EU** or
+  **non-EU**;
+- :guilabel:`Country Id`: must contain the country of the foreign seller in the two-letter ISO
+  (Alpha-2) code (ex., `IT` for 'Italy');
+- :guilabel:`CAP`: must be filled with `00000`;
+- :guilabel:`Partita Iva` (**VAT number**): must contain the **VAT** number for **EU businesses**
+  and `OO99999999999` (double 'O' **letter**, not 'zero') for **non-EU businesses**. In case of
+  private customers without **VAT** number, use `0000000`;
+- :guilabel:`Fiscal Code`: for foreign entities without an actual **Codice Fiscale**, any
+  recognizable identifier is valid.
+
+.. note::
+   Odoo does not support sending user-modified XML files.
```

Comment:
```suggestion
```

Why?

Reply:
We want to prevent to edit the XML and re-uploading it, right?
I know there are people who try to do that.

Reply:
We can say you can not upload XML files back into Odoo for sending if we say they can upload immediately to the government.